### PR TITLE
Add associations

### PR DESCRIPTION
**Pull request by HectorTorresE**
In this PR

- No linter errors
- Correct GitFlow used
- Class Classroom created with member variables label and students
- Relationships one-to-many created between Student and Classroom classes
- Class Book created with member variables title, author, and rentals
- Class Rental created with member variables date, book, and person
- Created relationships one-to-many between Person and Rental, and between Book and Rental

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (TedLivist) [inline book.rb:9]: - [ ] Your `:rentals` should have getter and setter, not just getter. Kindly fix that.

File: person.rb [hunk 2]
```diff
@@ -7,21 +8,27 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
     super
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
```
Comment (TedLivist) [inline person.rb:15]: - [ ] Your `rentals` should have the `attr_accessor`, not `attr_reader`. Kindly fix that.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @HectorTorresE,

Good job so far!
✔️ Good documentation
✔️ Implementation of associations
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 2 above)
Comment (HectorTorresE) [reply person.rb:15]: Thanks for the review, I will fix it ASAP.

File: book.rb (hunk 1 above)
Comment (HectorTorresE) [reply book.rb:9]: Thanks for the review, I will fix it ASAP.

**Review by mmsesay — APPROVED** [review]
Hi @HectorTorresE, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**